src/c4/bin/ythi.cc

```diff
@@ -0,0 +1,111 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   c4/bin/xthi.cc
+ * \author Mike Berry <mrberry@lanl.gov>, Kelly Thompson <kgt@lanl.gov>
+ * \date   Wednesday, Aug 09, 2017, 11:45 am
+ * \brief  Print MPI rank, thread number and core affinity bindings.
+ * \note   Copyright (C) 2017-2018 Los Alamos National Security, LLC.
```

Comment:
Please update the `\date`  field and limit the copyright to 2018 in the file header.